src/main/java/frc/robot/subsystems/Dashboard.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.subsystems;
```

Comment:
Are we missing any entries? I see the vision section is blank, but is there anything we need to add for it? Is there anything we need to add for any other subsystem?